### ISSUE TITLE
RPC: RPC_COULD_NOT_CONNECT: Could not connect

### ISSUE THREAD

**Issue by TheCoderSimp** [leonardssh/coc-discord-rpc#79]
### Operating system

WSL2 Ubuntu 20.04 LTS

### Neovim/Vim version

Neovim 0.8.3

### CoC version

0.0.82

### Discord version

official

### Reproduce step

I installed coc-discord-rpc using :CocInstall. It installed successfully but it instantly gave an error. I tried restarted nvim but it still didnt work

### Expected result

I expected it to connect to discord

### Actual result

Error message: `RPC: RPC_COULD_NOT_CONNECT: Could not connect

**Comment by TheWisePigeon** [comment]
Having the same issue, is WSL suppported at all?

**Comment by DoormatIka** [comment]
presence.nvim is also having this issue too: https://github.com/andweeb/presence.nvim/wiki/Rich-Presence-in-WSL

TLDR: If you're on WSL2, it's not gonna work.
